theme/static/js/bootstrap-formhelpers-custom.js

```diff
@@ -0,0 +1,24 @@
+/**
+* extend bootstrap-formhelpers.js v2.3.0 by @vincentlamanna
+* https://github.com/winmarkltd/BootstrapFormHelpers
+* modified here by Devin Cowan
+* because bootstrap-formhelpers appears to no longer be accepting PRs
+* a bug is patched directly in this copy
+* format can be null when "bfh-phone" field is used with a "data-country" attribute
+*/
+
+// TODO: remove notification
+alert("Note: your BS Formhelpers are bing overridden");
+
+let old_change = $.fn.bfhphone.prototype.change;
+$.fn.bfhphone.prototype.change = function(e) {
+  $this = $(this).data('bfhphone');
+  if ( typeof $this.options.format == 'undefined' ) return;
+  old_change.call(this, e);
+};
+
+let old_formatter = $.fn.formatNumber;
```

Comment:
@Maurier I'm not sure how to patch the `formatNumber` function from the bootstrap-form-helper library.

I think the issue is that the [formatNumber function](https://github.com/hydroshare/hydroshare/blob/4235-country-codes_s/theme/static/js/bootstrap-formhelpers.js#L16368) is defined in an [anonymous function](https://github.com/hydroshare/hydroshare/blob/4235-country-codes_s/theme/static/js/bootstrap-formhelpers.js#L16249) instead of being registered as a jQuery plugin (like the `$.fn.bfhphone.prototype.change` is). So I'm not sure how to access it to override it...any idea?

Reply:
What I'm doing here with `$.fn.formatNumber;` doesn't work.

Reply:
@devincowan In that case let's go with your original idea modifying the script itself. I don't think this dependency will be upgraded again.

Reply:
sounds good, thanks @Maurier, I reverted back to the original.

Reply:
Alright, let me know when it's ready for review.

Reply:
ready for review. though it is reverted back to what you reviewed the already ;)